scoda/templates/scoda.html

```diff
@@ -20,6 +24,75 @@
     <link rel="stylesheet" href="/static/dist/fonts/fontawesome-free/css/all.min.css" />
     <link href="https://fonts.googleapis.com/css2?family=Megrim&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="/static/dist/css/ds-style.css" />
+    <style>
```

Comment:
@riaansnyders please move these styles into their own sass partial, either new or existing. As a general rule we don't use inline styles, unless unavoidable.